darwin/cli.py

```diff
@@ -112,7 +112,7 @@ def _run(args: Namespace, parser: ArgumentParser) -> None:
         elif args.action == "pull":
             f.pull_dataset(args.dataset, args.only_annotations, args.folders, args.video_frames)
         elif args.action == "import":
-            f.dataset_import(args.dataset, args.format, args.files, args.append)
+            f.dataset_import(args.dataset, args.format, args.files, args.append, args.yes)
```

Comment:
Blocked for another reason:
```suggestion
            f.dataset_import(args.dataset, args.format, args.files, args.append, not args.yes)
```

In the other functions, we're using `class_prompt` as a flag that is `True` when `--yes` is `False`!

Luckily caught that today, when using this branch to upload data with classes which did not exist! Yesterday everything went fine as classes existed in the team, but not in the dataset, and were automatically added